include/cutlass/vector_types.h

```diff
@@ -31,13 +31,17 @@
 #pragma once
 
 #if defined(CUTLASS_ENABLE_SYCL)
-#include <sycl/sycl.hpp>
+#include "cutlass/detail/helper_macros.hpp"
```

Comment:
Should this be wrapped around #if defined(CUTLASS_ENABLE_SYCL) ? I think this can be outside of this if statement

Reply:
It is only needed for the code that is inside the `#if defined(CUTLASS_ENABLE_SYCL)`